vmdb/app/models/ems_refresh/save_inventory_infra.rb

```diff
@@ -298,4 +299,33 @@ def save_storage_files_inventory(storage, hashes)
     deletes = storage.storage_files(true).dup
     save_inventory_multi(:storage_files, storage, hashes, deletes, :name)
   end
+
+  def save_orchestration_stacks_inventory(ems, hashes, target = nil)
+    target = ems if target.nil?
+
+    ems.orchestration_stacks(true)
+    deletes = if (target == ems)
+                ems.orchestration_stacks.dup
+              else
+                []
+              end
+
+    _stacks = save_inventory_multi(:orchestration_stacks,
+                                   ems,
+                                   hashes,
+                                   deletes,
+                                   :ems_ref,
+                                   [:parameters])
+    store_ids_for_new_records(ems.orchestration_stacks, hashes, :ems_ref)
+  end
+
+  def save_parameters_inventory(orchestration_stack, hashes)
```

Comment:
Same reuse question here as for save_orchestration_stacks_inventory. It looks like we could just move the cloudy method to a common place (save_inventory_helper, or whatever) and reuse it.